packages/wrangler/src/config/environment.ts

```diff
@@ -1119,6 +1119,29 @@ export interface EnvironmentNonInheritable {
 		/** Whether the timer is enabled */
 		enable_timer?: boolean;
 	}[];
+
+	/**
+	 * Specifies rate limit bindings that are bound to this Worker environment.
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default []
+	 * @nonInheritable
+	 */
+	ratelimit: {
+		/** The binding name used to refer to the rate limiter in the Worker. */
+		name: string;
+		/** The namespace ID for this rate limiter. */
+		namespace_id: string;
```

Comment:
Where does this come from? Is there a CRUD API for creating these? If so, we should have Wrangler commands to speak to it (can be a followup)

Reply:
Is it a number or a string? This type seems to indicate a string